api/src/backend/tasks/tasks.py

```diff
@@ -152,3 +170,125 @@ def perform_scan_summary_task(tenant_id: str, scan_id: str):
 @shared_task(name="tenant-deletion")
 def delete_tenant_task(tenant_id: str):
     return delete_tenant(pk=tenant_id)
+
+
+def batched(iterable, batch_size):
+    """
+    Yield successive batches from an iterable.
+
+    Args:
+        iterable: An iterable source of items.
+        batch_size (int): The number of items per batch.
+
+    Yields:
+        tuple: A pair (batch, is_last_batch) where:
+            - batch (list): A list of items (with length equal to batch_size,
+              except possibly for the last batch).
+            - is_last_batch (bool): True if this is the final batch, False otherwise.
+    """
+    batch = []
+    for item in iterable:
+        batch.append(item)
+        if len(batch) == batch_size:
+            yield batch, False
+            batch = []
+
+    yield batch, True
+
+
+@shared_task(base=RLSTask, name="scan-output", queue="scans")
```

Comment:
We talk about `reports` everywhere. Since it is a `RLSTask`, it means this task can be checked by the user using the `/tasks` endpoints. Shouldn't we call it `scan-report`?

Also, the queue concerns me. Regarding Prowler Cloud, we do not have active workers for the `scans` queue. This means that we will have to adapt more logic to deploy another ECS for this kind of tasks.